src/components/field-card/FieldCard.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { SFieldCard } from './styles/SFieldCard';
+import { SFieldCardInnerRectangle } from './styles/SFieldCardInnerRectangle';
+import { SFieldCardText } from './styles/SFieldCardText';
+
+interface IFieldCard {
+  cardText?: string,
+}
+
+export const FieldCard: React.FC<IFieldCard> = ({ cardText }) => {
```

Comment:
@steiningermona As a patter whenever the component has an active state lets pass the active prop and an onClick handler to set it.